R/add_sample_metadata.R

```diff
@@ -0,0 +1,28 @@
+#' Add data frame with sample metadata to an SCE object
+#'
+#' @param sce SingleCellExperiment object to add metadata too
+#' @param metadata_df A dataframe with metadata to add to the sce.
+#'
+#' @return SingleCellExperiment object with a `metadata_df` added to the `sample_metadata`
+#'   slot of the metadata list
+#' @export
+#'
+add_sample_metadata <- function(sce,
+                                metadata_df){
```

Comment:
I think I was expecting this function to also check that the samples are present in the SCE, if we know that. I think we do with `metadata(sce)$sample_id`?